tests/pauli/dla/test_lie_closure.py

```diff
@@ -56,8 +56,10 @@ def test_init(self):
         vspace = PauliVSpace(ops1)
 
         assert all(isinstance(op, PauliSentence) for op in vspace.basis)
-        assert np.allclose(vspace._M, [[1.0, 1.0], [1.0, 0.0]]) or np.allclose(
-            vspace._M, [[1.0, 0.0], [1.0, 1.0]]
+        assert np.allclose(
+            vspace._M, [[1 / np.sqrt(2), 1.0], [1 / np.sqrt(2), 0.0]]
+        ) or np.allclose(
+            vspace._M, [[1 / np.sqrt(2), 0.0], [1 / np.sqrt(2), 1.0]]
```

Comment:
Is there a test we can add that fails on master but is fixed by this upgrade? :)

Reply:
Typically the singular matrix error was raised for larger systems that run for >2mins, I wouldnt want to add that to the test suite

Reply:
Mhm, that's a pity, but definitely agree here!